def test_exception():
  string1 = 'Hello'
  string2 = 'Test!'
  print(string1+strng2)